app/models/job.rb

```diff
@@ -5,10 +5,12 @@ class Job < ApplicationRecord
     mount_uploader :avatar, AvatarUploader
     has_one_attached :resume
     validates_length_of :title, maximum: 25, allow_blank: false
-    validates :description,presence: true
+    validates :description,:url,:email,:job_author,presence: true
     validates :url,presence: true, format: URI::DEFAULT_PARSER.make_regexp(%w[http https])
 
-    JOB_TYPES =["Full-time", "Part-time", "Contract", "Freelance"]
-
+    JOB_TYPES =["Full-time employment", "Part-time", "short-timeContract", "Freelance", "Internship","Volunteer contribution","Partner for a venture",]
+    CATEGORY  =["Programming","Interaction Design", "Graphic Design","Electronics","Information Security",
+                 "Testing/Quality Assurance","IT/System Administration","Business/Management",
+                "Writer/Editor","Customer Support","Mobile(iPhone,Android,Other)","Office Administation"]
```

Comment:
align these properly. All the items in the array should be visible without having to scroll horizontally

Reply:
alignment have been changed